tests/posix/common/src/pthread.c

```diff
@@ -812,3 +812,24 @@ ZTEST(posix_apis, test_pthread_dynamic_stacks)
 	zassert_ok(pthread_join(th, NULL));
 	zassert_equal(0xB105F00D, x);
 }
+
```

Comment:
Might use a #define value here to make it a bit more obvious what's being tested?

Reply:
Fixed!